tests/main/remodel-kernel/task.yaml

```diff
@@ -34,6 +38,22 @@ restore: |
         exit
     fi
 
+    # Wait for the system to stop processing stuff
+    # XXX: What was it processing? I got this error:
+    # error: snap "pc-kernel" has "refresh-snap" change in progress
+    while snap changes | grep Doing; do
```

Comment:
can we run this with retry-tool instead? I'm worried about this hanging forever it something goes wrong, especially since this is in the restore section.

Reply:
I've changed similar code in other branches into `snap watch --last=...` and I'm doing so for this one as well.